src/foam/core/notification/email/MicrosoftGraphEmailAgent.js

```diff
@@ -0,0 +1,326 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core.notification.email',
```

Comment:
I would rather put in a sub-package called 'ms' or something that could have its own POM and javaDependencies and journal files and be more self-contained, then those who don't use it could exclude or flag that POM off. And it could refine the EmailServiceConfig.